### PR TITLE
Refactor worldrenderer

### PR DESCRIPTION
**Pull request by Jurgenvs**
Can you run this on your android phone and see if it is functioning properly? Should now be able to switch between landscape and portrait modes. Do note: It is in 'fullUser' which means that if your android phone has screen orientation locked, so will the game.

### CODE REVIEW

**Comment by Surskit** [comment]
Missing Javadoc in some places, otherwise looks good and works correctly on my phone aswell.

File: core/src/cg/group4/view/screen/HomeScreen.java [hunk 1]
```diff
@@ -76,11 +88,17 @@ public final void initHomeScreenTitle() {
      * If the timer is started, its label will change to the time left on the timer.
      */
     public final void initStrollIntervalTimer() {
-        timer = new Label("3600", cGameSkin.get("default_labelStyle", Label.LabelStyle.class));
+        cIntervalTimer = TimeKeeper.getInstance().getTimer(Timer.Global.INTERVAL.name());
+        System.out.println(cIntervalTimer.getRemainingTime() + "");
+        if(cIntervalTimer.getRemainingTime() > 0 ){
```
Comment (foresterre) [inline core/src/cg/group4/view/screen/HomeScreen.java]: Needs whitespace for better readability.

File: core/src/cg/group4/view/screen/HomeScreen.java [hunk 2]
```diff
@@ -76,11 +88,17 @@ public final void initHomeScreenTitle() {
...
+            cTimer = new Label(Integer.toString(cIntervalTimer.getRemainingTime()), cGameSkin.get("default_labelStyle", Label.LabelStyle.class));
+        } else {
+            cTimer = new Label("Ready", cGameSkin.get("default_labelStyle", Label.LabelStyle.class));
+        }
         TimeKeeper.getInstance().getTimer(Timer.Global.INTERVAL.name()).subscribe(new TimerTask() {
```
Comment (foresterre) [inline core/src/cg/group4/view/screen/HomeScreen.java]: Needs whitespace for better readability.

File: core/src/cg/group4/view/screen_mechanics/WorldRenderer.java [hunk 3]
```diff
@@ -13,173 +11,246 @@
...
+    /**
+     * Discards previous background textures (if existing) and then sets the background to the file in the FileHandle.
+     * If the new background does not fit the aspect ratio, the default background will be set.
+     * @param fileHandle The FileHandle to the file.
+     */
+    public void setBackground(FileHandle fileHandle){
+        if(cBackgroundSprite != null){
+            cBackgroundSprite.getTexture().dispose();
+        }
+        Texture texture = new Texture(fileHandle);
```
Comment (foresterre) [inline core/src/cg/group4/view/screen_mechanics/WorldRenderer.java]: Needs whitespace for better readability.

**Comment by Jurgenvs** [comment]
Please check again if there are any problems. Would like to merge ASAP to prevent branches stacking up.

**Comment by foresterre** [comment]
Its looking good!

**Pull request merged**